### DISCUSSION TITLE
How to set the location of terminal startup?

### DISCUSSION THREAD

**Discussion by xiaofanshifu** [microsoft/terminal#12998] [Q&A] [answered]
Can I set the location of terminal startup in "Settings"?
![image](https://user-images.githubusercontent.com/57950591/165872075-b52f921c-fd82-447a-8bed-23918433889c.png)

settings.json
![image](https://user-images.githubusercontent.com/57950591/165874190-784dba09-fc4d-47a9-8a61-1c4477f4df96.png)

**Comment by zadjii-msft** [comment, marked as answer]
Alas, this is one of the few settings that doesn't have a Settings UI equivalent yet. See: #9075
